cfme/web_ui/paginator.py

```diff
@@ -1,80 +1,49 @@
 """A set of functions for dealing with the paginator controls."""
-from cfme.web_ui import Select, Input, AngularSelect
-import cfme.fixtures.pytest_selenium as sel
-import re
-from selenium.common.exceptions import NoSuchElementException
-from functools import partial
-from utils import version
 from cfme.exceptions import PaginatorException
+from widgetastic_manageiq import PaginationPane
+from utils.appliance import get_or_create_current_appliance
 
-
-def _locator():
-    return version.pick({version.LOWEST: '(//div[@id="paging_div"]//div[@id="pc_div_1"])',
-        '5.5': '(//div[@id="paging_div"]//div[@id="rpb_div_1" or @id="pc_div_1"])'})
-
-
-_next = '//img[@alt="Next"]|//li[contains(@class, "next")]/span'
-_previous = '//img[@alt="Previous"]|//li[contains(@class, "prev")]/span'
-_first = '//img[@alt="First"]|//li[contains(@class, "first")]/span'
-_last = '//img[@alt="Last"]|//li[contains(@class, "last")]/span'
-_num_results = '//select[@id="ppsetting" or @id="perpage_setting1"]'
-_sort_by = '//select[@id="sort_choice"]'
-_page_cell = '//td//td[contains(., " of ")]|//li//span[contains(., " of ")]'
-_check_all = Input("masterToggle")
-
-_prefix = r"(?:Items?|Rows?|Showing)?\s*"
-_regexp = r"{}(?P<first>\d+)-?(?P<last>\d+)? of (?P<total>\d+)\s*(?:items?)?".format(_prefix)
+appliance = get_or_create_current_appliance()
```

Comment:
this bit shouldn't be here, import time obtaining of the appliance and using of a object that may be dead

i suspect its possible that this one starts a first browser artifactor with a wrong artifactor server url

later on we start the real one and the issue gets hidden